src/LaunchDarkly.Client/FeatureRequestor.cs

```diff
@@ -12,11 +12,16 @@ namespace LaunchDarkly.Client
 {
     internal class FeatureRequestor
     {
+
+        internal struct VersionedFeatureFlags
+        {
+            public IDictionary<string, FeatureFlag> FeatureFlags { get; set; }
+            public string VersionIdentifier { get; set; }
```

Comment:
What exactly is this VersionIdentifier? Some comments on this struct could be helpful. Keep in mind that We already have a notion of a Version associated with a FeatureFlag- it is numeric.